src/components/Form/FormSection.jsx

```diff
@@ -5,7 +5,6 @@ import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
```

Comment:
`FormSection` will also be deemed completely unnecessary and deleted once PR #388 gets merged in, and this one gets merged in.  It's pretty much unnecessary without the old notification system. Doesn't really add anything.  We can copy the `LogoutModal` pretty much where the title is in `<DialogTitle />` of `<Dialog />`.

Reply:
This means I don't need to delete it now then? Trying to determine if this is an actionable comment.

Reply:
I would still make the changes you did for now... Just kind of depends on whose PR gets merged first I guess. If Brandon's, then I think this whole file can be deleted.